userspace/libscap/examples/01-open/scap_open.c

```diff
@@ -150,6 +150,11 @@ static int simple_set[] = {
 	-1
 };
 
+typedef struct ppm_sc_counter{
+	uint64_t counter;
+	ppm_sc_code code; /* we need the code also here because at the end we will sort */
```

Comment:
Why using ppm_sc_code and not PPME?
I mean, ppm_sc_code do not map enter/exit events

Reply:
I used ppm_sc to obtain information also on generic syscalls, otherwise, we would have just thousands of `PPME_GENERIC_E/X` without knowing the underlying syscall. The approach is slightly more complex and a little bit less efficient but in the end provides more information that could be interesting... since this tool is not run in production it seems a great compromise to me

Reply:
ah i see, makes sense!